### DISCUSSION TITLE
mspider/user-guide/sidecar-management/global-sidecar/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1168] [Announcements]
# mspider/user-guide/sidecar-management/global-sidecar/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/sidecar-management/global-sidecar/

**Comment by liyuerich** [comment]
边车管理 -> 全局边车管理
边车管理中 没有 全局边车管理

**Comment by windsonsea** [comment]
Fixed in PR #1169. Only an external mesh can support this global sidecar settings.
